webpack/scenes/AlternateContentSources/MainTable/ACSTable.js

```diff
@@ -389,9 +404,13 @@ const ACSTable = () => {
                       emptyMessage="N/A"
                     />
                     </Td>
-                    <Td isActionCell>
-                      <ActionsColumn items={rowDropdownItems(acs)} />
-                    </Td>
+                    {(hasPermission(permissions, 'destroy_alternate_content_sources') ||
```

Comment:
why can't you use `canDelete` and `canEdit` here?

Reply:
CanDelete and canEdit are object level. The hasPermissions will check record level permissions for the row actions which is more accurate.